src/process_impl/builder.rs

```diff
@@ -0,0 +1,92 @@
+use super::super::SessionBuilder;
+use super::{Error, Session};
+
+use std::path::Path;
+use std::process::Stdio;
+
+use tempfile::Builder;
+use tokio::io::AsyncReadExt;
+use tokio::process;
+
+pub(crate) async fn just_connect<S: AsRef<str>>(
```

Comment:
It's hard to tell at a glance — are there any differences in the body of this function from what it used to be before this PR?

Reply:
Nah, I copied them without any change.